### DISCUSSION TITLE
Filename database identifiers

### DISCUSSION THREAD

**Discussion by sundermann** [Komet/MediaElch#1652] [Q&A]
Does Mediaelch support reading database identifiers from the filename/path as implemented in [Jellyfin](https://jellyfin.org/docs/general/server/media/movies/), [Emby](https://emby.media/support/articles/Movie-Naming.html#id-tags-in-folder--file-names), and [Plex](https://support.plex.tv/articles/naming-and-organizing-your-tv-show-files/)?

**Comment by bugwelle** [comment]
Hi,

no, MediaElch does not support such a feature. While you can rename your movies to include such IDs (using MediaElch's renamer), MediaElch does not respect/use the IDs in filenames when loading an initial list of movies. IDs are only used/loaded when set in a `movie.nfo`.
